solidity/src/base/Errors.sol

```diff
@@ -61,6 +61,8 @@ uint32 constant ERR_BIT_DECOMPOSITION_INVALID = 0xda443b2b;
 uint32 constant ERR_INVALID_VARYING_BITS = 0x76d56a3d;
 /// @dev Error code for when monotony check fails.
 uint32 constant ERR_MONOTONY_CHECK_FAILED = 0x976f97b8;
+/// @dev Error code for when bits that shouldn't vary do vary.
+uint32 constant ERR_MISMATCHING_COLUMN_AND_CANDIDATE_LENGTHS = 0x32e88b78;
```

Comment:
You mean number of columns? "Length" is very easily misunderstood as column length.